pandas/_libs/tslibs/frequencies.pxd

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-cpdef object get_rule_month(object source, object default=*)
+cpdef str get_rule_month(object source, str default=*)
```

Comment:
Do we get any performance boosts from the stronger types here?

Reply:
None that I can detect with timeit